bundles/specmate-emfjson/src/com/specmate/emfjson/EMFJsonSerializer.java

```diff
@@ -190,7 +191,8 @@ private Object serializeValue(Object value) throws SpecmateException {
 		} else if (value instanceof Boolean) {
 			return value;
 		} else {
-			return value.toString();
+			String v = value.toString();
+			return Encode.forHtml(v);

```

Comment:
If a node contains a '<' character, like _the age < 18_ the '<' is replaced as well. This shouldn't be